CHANGELOG.md

```diff
@@ -17,6 +17,7 @@
 - new Dart code file structure #96 
 - Base the sdk name on the platform (`sentry.dart` for io & flutter, `sentry.dart.browser` in a browser context) #103 
 - Single changelog and readme for both packages #105
+- fix a test regression introduced by #103 #105
```

Comment:
I'd suggest keeping the PR's title and changelog aligned.

if I read the PR's title, it doesn't look like it's fixing a test regression.

Reply:
ok now I see it: based on the **test platform**